builder/js/ncf.js

```diff
@@ -806,6 +806,12 @@ var SaveChangesModalCtrl = function ($scope, $modalInstance, technique, editForm
   };
 };
 
-app.config(function($locationProvider) {
-    $locationProvider.html5Mode(false).hashPrefix('!');;
+app.config(function($httpProvider,$locationProvider) {
+    $locationProvider.html5Mode(false).hashPrefix('!');
+    if (!$httpProvider.defaults.headers.get) {
+        $httpProvider.defaults.headers.get = {};    
+    }    
+    $httpProvider.defaults.headers.get['If-Modified-Since'] = 'Mon, 26 Jul 1997 05:00:00 GMT';
```

Comment:
Why that date ? can you add a comment to explain all those headers ? (or at leas this one!)